Project.toml

```diff
@@ -1,4 +1,4 @@
-name = "JSX"
+name = "HyperscriptLiteral"
```

Comment:
We might change the name again (HyperscriptLiteral.jl is too similar to other packages). 🙈